### PR TITLE
Improve CLI capsule tests docstrings

### PR DESCRIPTION
**Pull request by elyezer**
Move the test modules to the right location. Improve docstrings
formatting and use `@test` as first docstring's line.

Also import CLITestCase which is the base test case class.

This is a follow up for the #1825

### CODE REVIEW

File: tests/foreman/cli/test_capsule.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        @Feature: Capsules
+
+        @Setup: Some valid, functional compute resource
+        (perhaps one variation of this case for each supported compute
+        resource type).  Also, functioning capsule with proxy is required.
+
+        @Steps:
+            1. Attempt to route provisioning content through capsule that is
+            using a proxy
+            2. Attempt to provision instance
```
Comment (Ichimonji10) [inline tests/foreman/cli/test_capsule.py]: Try executing `make docs` (or is it `make docs-html`?). This should throw a warning.

Comment (Ichimonji10) [inline tests/foreman/cli/test_capsule.py]: If a numbered list is being created here, then the list should be separated from `@steps:` by a blank line and the list should be un-indented.

When a reST parser encounters an indented block of text, it expects that that indented block of text will be preceded by e.g. double colons (`::`), denoting that the indented block is code.

File: tests/foreman/cli/test_capsule.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# -*- encoding: utf-8 -*-
+# vim: ts=4 sw=4 expandtab ai
```
Comment (Ichimonji10) [inline tests/foreman/cli/test_capsule.py]: Is there a good reason to set the `autoindent` option? There are several indentation modes, and overriding user preferences seems like a bad idea.

**Comment by Ichimonji10** [comment]
Both of the comments given apply in more than one location.

ACK pending comments.

**Comment by elyezer** [comment]
Comments addressed, the editor line comment was dropped. Enforcing a editor config is not good and that line is better used for files which the editor does not recognize by default the filetype.

**Comment by cswiii** [comment]
ACK

File: docs/api/tests.foreman.cli.rst [hunk 3]
```diff
@@ -19,6 +19,20 @@
...
+    :members:
+    :undoc-members:
+
+:mod:`tests.foreman.cli.test_capsule_installer`
+-----------------------------------------------
+
+.. automodule:: tests.foreman.cli.test_capsule_installer
+    :members:
+    :undoc-members:
+
```
Comment (Ichimonji10) [inline docs/api/tests.foreman.cli.rst]: As a follow-up pull request, we should make use of the `autodoc_default_flags` configuration option. See [here](https://github.com/SatelliteQE/nailgun/commit/b8fc67a9b75d807aae11185362ce21d02939fc57).

File: tests/foreman/cli/test_capsule.py [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+        @Steps:
+        1. attempt to subscribe a system to a content type variation,
+        via a proxy-enabled capsule
+
+        @Assert: system is successfully subscribed to each content type
+
+        @Status: Manual
+
+        """
+    def consume_content_through_capsule(self):
```
Comment (Ichimonji10) [inline tests/foreman/cli/test_capsule.py]: Add a blank line directly above this method declaration?

Comment (elyezer) [inline tests/foreman/cli/test_capsule.py]: Travis does not get this :disappointed:

File: docs/api/tests.foreman.cli.rst (hunk 3 above)
Comment (elyezer) [inline docs/api/tests.foreman.cli.rst]: Agreed

**Comment by Ichimonji10** [comment]
Looks great. Thank you @elyezer. ACK

**Pull request merged**